devops/smart-install.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+changedFiles="$(git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD)"
```

Comment:
This will work only if you're having the latest master fetched, and you're not on the master commit.